<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RequestForm</class>
 <widget class="QDialog" name="RequestForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Request Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
background: #8B0000;

}</string>
  </property>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
image: url(&quot;C:/Users/clint/Pictures/sblood.png&quot;);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>SureNeed</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Patient Clinical Record</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtMiddleName">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>150</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border-bottom: 1px solid black;
background: transparent;
}
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtGender">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>220</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border-bottom: 1px solid black;
background: transparent;
}
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDateofBirth">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>280</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border-bottom: 1px solid black;
background: transparent;
}
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtContactNumber">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>340</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border-bottom: 1px solid black;
background: transparent;
}
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtOccupation">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>400</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border-bottom: 1px solid black;
background: transparent;
}
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtDateRequested">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>460</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
border-bottom: 1px solid black;
background: transparent;
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>460</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Date Requested:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>452</x>
     <y>400</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Occupation:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>422</x>
     <y>340</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Contact Number:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>442</x>
     <y>280</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Date of Birth:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>220</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Gender:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>442</x>
     <y>150</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Middle Name:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>540</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtCID">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>90</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>90</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>C-ID:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>761</width>
     <height>391</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{
border: 1px solid black;
}</string>
   </property>
   <property name="title">
    <string>FORM</string>
   </property>
   <widget class="QLineEdit" name="txtBloodTypeNeeded">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>340</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
border-bottom: 1px solid black;
background: transparent;
}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Blood Type Needed:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtEmailAddress">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>280</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
border-bottom: 1px solid black;
background: transparent;
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtAddress">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>220</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
border-bottom: 1px solid black;
background: transparent;
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtAge">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>160</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
border-bottom: 1px solid black;
background: transparent;
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtLastName">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
border-bottom: 1px solid black;
background: transparent;
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtFirstName">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
border-bottom: 1px solid black;
background: transparent;
}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>First Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Last Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Age:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>220</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Address:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>280</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Email Address:</string>
    </property>
   </widget>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>label_13</zorder>
  <zorder>label_12</zorder>
  <zorder>label_14</zorder>
  <zorder>txtMiddleName</zorder>
  <zorder>txtGender</zorder>
  <zorder>txtDateofBirth</zorder>
  <zorder>txtContactNumber</zorder>
  <zorder>txtOccupation</zorder>
  <zorder>txtDateRequested</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>pushButton</zorder>
  <zorder>txtCID</zorder>
  <zorder>label_16</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
